packages/sdk/src/send/send.js

```diff
@@ -4,11 +4,16 @@ import {resolve as defaultResolve} from "../resolve/resolve.js"
 import {send as defaultSend} from "./sdk-send.js"
 
 export const send = async (args = [], opts = {}) => {
-  const sendFunction = await config().get("sdk.send", opts.send || defaultSend)
-  const resolveFunction = await config().get(
-    "sdk.resolve",
+  const sendFn = await config.first(
```

Comment:
Should we update [packages/fcl/src/serialize/index.js](https://github.com/onflow/flow-js-sdk/blob/qvvg/config-all/packages/fcl/src/serialize/index.js) to use new `config.first` in this release?

```
export const serialize = async (args = [], opts = {}) => {
  const resolveFunction = await config().get(
    "sdk.resolve",
    opts.resolve || defaultResolve
  )
  ...  
}
```

Reply:
Yeah, can do that in a different PR though.

Reply:
Should probably move it into the sdk too